app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsFragment.java

```diff
@@ -97,6 +97,8 @@ public void onStop() {
     }
 
     public void openCreateTicketFragment() {
+        getPresenter().isNewTicketCreated = true;
```

Comment:
Why can't you set it from inside the presenter?

Reply:
I want to set it only when a new ticket is created. How to do it from inside the `TicketsPresenter`?

Reply:
Hmm, openCreateTicketFragment doesn't mean that ticket will be created

Reply:
Yes I know but then should we set it from `CreateTicketPresenter`?

Reply:
Try

Reply:
Updated.